addons/website_blog/models/website_blog.py

```diff
@@ -194,7 +194,7 @@ def _set_post_date(self):
                 blog_post._write(dict(post_date=blog_post.create_date)) # dont trigger inverse function
 
     def _check_for_publication(self, vals):
-        if vals.get('website_published'):
+        if vals.get('is_published'):
```

Comment:
In this case it is fine checking only `is_published` as this method is call from write & create but if calling write or create with `website_published`, it will end up doing another write with `is_published` ('related').